ocaml/parsing/jane_syntax.ml

```diff
@@ -142,17 +165,17 @@ module Comprehensions = struct
 
   let expr_of ~loc cexpr =
     (* See Note [Wrapping with make_entire_jane_syntax] *)
-    AST.make_entire_jane_syntax Expression ~loc extension_string (fun () ->
+    AST.make_entire_jane_syntax Expression ~loc erasability feature (fun () ->
       match cexpr with
       | Cexp_list_comprehension comp ->
           expr_of_comprehension ~type_:["list"] comp
       | Cexp_array_comprehension (amut, comp) ->
           expr_of_comprehension
             ~type_:[ "array"
-                   ; match amut with
-                     | Mutable   -> "mutable"
-                     | Immutable -> "immutable"
-                   ]
+                  ; match amut with
+                    | Mutable   -> "mutable"
+                    | Immutable -> "immutable"
+                  ]
```

Comment:
```suggestion
                   ; match amut with
                     | Mutable   -> "mutable"
                     | Immutable -> "immutable"
                   ]
```

Why did this get outdented?  Is there a style rule I don't know?

Reply:
Nope, just an artifact of a draft of this where this changed indentation level.